.devcontainer/initialize-command.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Immediately catch all errors
+set -eo pipefail
+
+# Uncomment for debugging
+# set -x
+# env
+
+# Use first argument as target name
+target=$1
+
+################################################################################
+# MARK: Pull image - download image from CI and GHCR for local dev container
+# REFERENCE_IMAGE=ghcr.io/ros-navigation/navigation2:main-debugger
+# docker pull $REFERENCE_IMAGE
+# export DEV_FROM_STAGE=$REFERENCE_IMAGE
```

Comment:
To create a dev container using the CI images from this PR, including a pre-built colcon workspace, simply uncomment the lines above and change the following tagname to match the current branch before using dev container tooling to create the container.

```diff
-REFERENCE_IMAGE=ghcr.io/ros-navigation/navigation2:main-debugger
+REFERENCE_IMAGE=ghcr.io/ros-navigation/navigation2:buildkit-debugger
```

Reply:
For more information in getting started, check the included `.devcontainer/README.md` for details.